app/lib/forms/choose_private_childcare_provider.rb

```diff
@@ -4,10 +4,9 @@ class ChoosePrivateChildcareProvider < Base
 
     attr_accessor :institution_name, :institution_identifier
 
-    validates :institution_identifier, format: { with: /\APrivateChildcareProvider-.+\z/, unless: -> { institution_identifier.blank? || institution_identifier == "other" } }
```

Comment:
Do we need to do this in Forms::ChooseSchool and Forms::ChooseChildcareProvider?

Reply:
I don't believe so, in the locales it's that the generic `invalid` is paired up with `%{urn}` that causes the issue.

```yml
invalid: No private childcare providers with the URN %{urn} were found, please try again
```

There probably is scope to make sure the generic error messages are nicely worded but I think that should be a separate piece of work.